src/libsyntax/print/pprust.rs

```diff
@@ -2607,13 +2605,8 @@ impl<'a> State<'a> {
     }
 
     pub fn print_fn_output(&mut self, decl: &ast::FnDecl) -> IoResult<()> {
-        if let ast::Return(ref ty) = decl.output {
-            match ty.node {
-                ast::TyTup(ref tys) if tys.is_empty() => {
-                    return self.maybe_print_comment(ty.span.lo);
```

Comment:
Just making sure, but was it intentional to lose the `maybe_print_comment`? (not that I know what it does...)